x-pack/plugins/uptime/server/lib/requests/get_latest_monitor.ts

```diff
@@ -53,11 +55,10 @@ export const getLatestMonitor: UMElasticsearchQueryFn<GetLatestMonitorParams, Pi
   };
 
   const result = await callES('search', params);
-  const ping: any = result.hits?.hits?.[0] ?? {};
-  const { '@timestamp': timestamp, ...monitorResult } = ping?._source ?? {};
 
-  return {
-    timestamp,
-    ...monitorResult,
-  };
+  const decoded = PingType.decode(result.hits?.hits?.[0]?._source ?? {});
+  if (isRight(decoded)) {
+    return decoded.right;
+  }
+  throw new Error(JSON.stringify(PathReporter.report(decoded)));
```

Comment:
i think we can wrap this as part of Create Route Wrapper for all API's just like front end, so just provide Type and it should auto check it against result.

Reply:
We should just add that work to https://github.com/elastic/kibana/issues/60395, as it's so closely related, rather than modifying the wrapper in this PR. Are you ok with punting on introducing that abstraction?